vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridMultiSelectionColumnPageIT.java

```diff
@@ -64,6 +64,30 @@ public void selectAllCheckbox() {
                 "Selected item count: "
                         + (GridMultiSelectionColumnPage.ITEM_COUNT - 1),
                 message.getText());
+
+        // Test switch selection mode from single to multi mode before adding the grid to DOM
+        WebElement gridSelectionMode = findElement(By.id("in-testing-multi-selection-mode-grid"));
```

Comment:
This test is already big enough. This code should be in a new test method.

Reply:
done